db/seeds.rb

```diff
@@ -10,3 +10,112 @@
+                  genre: 'drama',
+                  duration: 210 },
+                { title: 'Pulp Fiction',
+                  description: "In the realm of underworld, a series of incidents intertwines the lives
+                                of two Los Angeles mobsters, a gangster's wife, a boxer and two small-time criminals.",
+                  director: 'Quentin Tarantino',
+                  genre: 'crime',
+                  duration: 178 },
+                { title: 'Forrest Gump',
+                  description: 'The presidencies of Kennedy and Johnson, the Vietnam War, the Watergate scandal
+                                and other historical events unfold from the perspective of an Alabama man with
+                                an IQ of 75, whose only desire is to be reunited with his childhood sweetheart.',
+                  director: 'Robert Zemeckis',
+                  genre: 'drama',
+                  duration: 142 },
+                { title: 'Gladiator',
+                  description: 'A former Roman General sets out to exact vengeance against the corrupt
+                                emperor who murdered his family and sent him into slavery.',
+                  director: 'Ridley Scott',
+                  genre: 'historical drama',
+                  duration: 171 }
+              ])
+Rails.logger.info 'Movies created.'
+
+Rails.logger.info 'Creating Ticket Desks...'
+ticket_desk = TicketDesk.create!
+Rails.logger.info 'Ticket Desks created.'
+
+Rails.logger.info 'Creating Screenings...'
+Screening.create!([
+                    { cinema_hall: CinemaHall.all.sample, screening_time: DateTime.now + 1.hour,
+                      movie: Movie.all.sample },
+                    { cinema_hall: CinemaHall.all.sample, screening_time: DateTime.now + 2.hour,
+                      movie: Movie.all.sample },
+                    { cinema_hall: CinemaHall.all.sample, screening_time: DateTime.now + 3.hour,
+                      movie: Movie.all.sample },
+                    { cinema_hall: CinemaHall.all.sample, screening_time: DateTime.now + 4.hour,
+                      movie: Movie.all.sample }
+                  ])
+Rails.logger.info 'Screenings created.'
+
+Rails.logger.info 'Creating Reservations...'
+Reservation.create!([
+                      { status: 'pending', screening: Screening.all.sample, ticket_desk: ticket_desk, user: client },
+                      { status: 'pending', screening: Screening.all.sample, ticket_desk: ticket_desk, user: client  },
+                      { status: 'pending', screening: Screening.all.sample, ticket_desk: ticket_desk, user: client  }
+                    ])
+Rails.logger.info 'Reservations created.'
+
+Rails.logger.info 'Creating Seats Reservations...'
+letters = ('A'..'J').to_a
+Reservation.all.each do |reservation|
+  rand(1..3).times do |_number|
+    SeatsReservation.create!([row: letters[rand(1..reservation.screening.cinema_hall.row_number) - 1],
```

Comment:
If you're looking for an easy option to get random numbers from a predefined set without repeating you can try [this simple solution](https://stackoverflow.com/questions/9457841/generate-non-repeating-random-numbers-in-ruby#answer-9458016).